actions/recipe_action.go

```diff
@@ -116,7 +119,7 @@ func (recipe *RecipeAction) Run(context *debos.DebosContext) error {
 	recipe.LogStart()
 
 	for _, a := range recipe.Actions.Actions {
-		if err := a.Run(context); err != nil {
+		if err := a.Run(&recipe.context); err != nil {
```

Comment:
This is too little; another Run action can change e.g. the rootdir which I don't think you'd catch this way